pkg/status/status.go

```diff
@@ -356,20 +355,7 @@ func expvarStats(stats map[string]interface{}) (map[string]interface{}, error) {
 	json.Unmarshal(forwarderStatsJSON, &forwarderStats) //nolint:errcheck
 	stats["forwarderStats"] = forwarderStats
 
-	runnerStatsJSON := []byte(expvar.Get("runner").String())
-	runnerStats := make(map[string]interface{})
-	json.Unmarshal(runnerStatsJSON, &runnerStats) //nolint:errcheck
-	stats["runnerStats"] = runnerStats
-
-	autoConfigStatsJSON := []byte(expvar.Get("autoconfig").String())
-	autoConfigStats := make(map[string]interface{})
-	json.Unmarshal(autoConfigStatsJSON, &autoConfigStats) //nolint:errcheck
-	stats["autoConfigStats"] = autoConfigStats
-
-	checkSchedulerStatsJSON := []byte(expvar.Get("CheckScheduler").String())
-	checkSchedulerStats := make(map[string]interface{})
-	json.Unmarshal(checkSchedulerStatsJSON, &checkSchedulerStats) //nolint:errcheck
-	stats["checkSchedulerStats"] = checkSchedulerStats
+	collector.UpdateStatus(stats)
```

Comment:
I find the name of this method a bit confusing. The other method is `GetStatus`, which is clearly a reader that returns data. Since `UpdateStatus` is ambiguous, at first read I keep thinking that this is taking an argument and writing it somewhere, rather than modifying its argument. One thought would be to change this to `InsertStatus` or `PopulateStatus`, but I'm open to other names that clarify how this method is adding more key-values into the map that gets passed to it.

Reply:
I like `PopulateStatus`